webpack/assets/javascripts/services/statisticsChartService.js

```diff
@@ -21,6 +22,37 @@ function getChartConfig(details) {
   return config;
 }
 
+function getChartConfigByData(_data, id, type) {
+  const filteredData = _data.filter(d => d[1] !== 0);
+  const data = filteredData.reduce(
+    (curr, next) => {
+      const key = next[0];
+      const names = {
+        ...curr.names,
+        [key]: key.replace(/_/g, ' ')
```

Comment:
why is this needed? i assume the server can provide the correct answer without _ ?